lib/util/git/index.ts

```diff
@@ -893,3 +896,30 @@ export function getUrl({
     pathname: repository + '.git',
   });
 }
+
+export async function pushCommitAsRef(
+  commitSha: string,
+  refName: string
+): Promise<void> {
+  await git.raw(['update-ref', refName, commitSha]);
+  await git.raw(['push', '--force', 'origin', refName]);
+}
+
+const treeItemRegex = regEx(
```

Comment:
should we create a `./util.ts` and move there, as this file is already pretty big?

Reply:
Then we should do something with internal `simple-git` instance in `index.ts`

Reply:
Let's not delay further if this is a major refactor

Reply:
I only meant the Regex constant, not the functions. 🤗